package.json

```diff
@@ -33,35 +33,39 @@
     "storage",
     "json"
   ],
-  "main": "lib/levelup.js",  
+  "main": "lib/levelup.js",
+  "typings": "lib/levelup.js",
   "dependencies": {
     "deferred-leveldown": "~2.0.0",
     "level-errors": "~1.1.0",
     "level-iterator-stream": "~2.0.0",
     "xtend": "~4.0.0"
   },
   "devDependencies": {
-    "abstract-leveldown": "^2.7.0",
+    "@types/node": "^8.0.31",
+    "abstract-leveldown": "^2.7.1",
     "after": "^0.8.2",
     "async": "^2.5.0",
     "bl": "^1.2.1",
     "browserify": "^14.3.0",
     "bustermove": "~1.0.0",
     "delayed": "~1.0.1",
-    "encoding-down": "^2.2.0",
+    "encoding-down": "^2.3.1",
     "faucet": "~0.0.1",
-    "leveldown": "^1.8.0",
-    "memdown": "^1.2.4",
+    "leveldown": "^2.0.0",
+    "memdown": "^1.4.1",
     "msgpack-js": "~0.3.0",
     "referee": "~1.2.0",
     "rimraf": "^2.6.1",
     "safe-buffer": "^5.1.0",
     "slow-stream": "0.0.4",
     "standard": "^10.0.2",
-    "tape": "^4.7.0"
+    "tape": "^4.7.0",
+    "ts-node": "^3.3.0",
+    "typescript": "^2.5.2"
   },
   "scripts": {
-    "test": "standard && node test | faucet" 
+    "test": "standard && ts-node --no-cache test | faucet"
```

Comment:
I think it should be *in addition to* node tests. Tests passing in TS is not a guarantee for tests passing in node.

Reply:
Sure, I can switch back to having both.